postgresql/files/restore.sh

```diff
@@ -33,11 +33,20 @@ touch /root/postgresql/flags/{{ database.name }}-installed
 {%- set dest_name = db_name + ".pg_dump.gz" %}
 {%- set target = "/root/postgresql/data/" %}
 
-scp backupninja@{{ database.initial_data.source }}:/srv/backupninja/{{ host }}/var/backups/postgresql/postgresql.{{ age }}/{{ source_name }} {{ target }}{{ dest_name }} 
+{%- if database.initial_data.auth is defined %}
+export AWS_ACCESS_KEY_ID={{ database.initial_data.auth.awsaccesskeyid }}
+export AWS_SECRET_ACCESS_KEY={{ database.initial_data.auth.awssecretaccesskey }}
+{%- endif %}
 
+{%- if database.initial_data.engine == "backupninja" %}
+scp backupninja@{{ database.initial_data.source }}:/srv/backupninja/{{ host }}/var/backups/postgresql/postgresql.{{ age }}/{{ source_name }} {{ target }}{{ dest_name }} 
```

Comment:
Backupninja doesn't mean it's using ssh to backup/restore + hard-coded path, etc. This is wrong.